go/internal/indexerdb/p2e.go

```diff
@@ -3,8 +3,10 @@ package indexerdb
 // Delete the record when user unstake
 type P2eSquadStaking struct {
 	OwnerID   UserID `gorm:"primaryKey"`
-	StartTime uint64 `gorm:"index"`
+	StartTime uint64 `gorm:"primaryKey"`
+	SeasonID  string `gorm:"index"`
 	EndTime   uint64 `gorm:"index"`
+	TokenIDs  string
```

Comment:
shouldn't the primary key also include the `TokenIDs`?
this can break if the `StartTime` is the same for two squads, no?

Reply:
From our dapp, with 1 ownerID we cannot have the same StartTimes for 2 different stakes but theorically a malicious user can build the messages him self and send 2 stakes in the same block so you're right. I will add TokenIDs in compound keys to be sure.